tests/performance/smoke.py

```diff
@@ -9,6 +11,12 @@ class SmokeBehavior(TaskSet):
     See https://bugzil.la/1186085
     """
 
+    def on_start(self):
+        locust_host = 'developer.allizom.org'
+        if 'LOCUST_HOST' in os.environ:
+            locust_host = os.environ['LOCUST_HOST']
```

Comment:
Let's do this in one line: `locust_host = os.environ.get('LOCUST_HOST', 'developer.allizom.org')`